backend/content-type/eCampMultiSelect/src/Strategy.php

```diff
@@ -19,11 +19,11 @@ public function __construct(OptionService $optionService, ServiceUtils $serviceU
         parent::__construct($serviceUtils);
 
         $this->optionService = $optionService;
+        $this->optionService->setEntityClass(Option::class);
```

Comment:
Wieso können das diese Services nicht selber wissen? Scheint mir nicht die Verantwortung des Callers, dem OptionService zu sagen dass er mit Options zu tun hat. Dieser kann ja nicht mit mehreren Entities arbeiten oder?

Reply:
Ja, find ich auch nicht so toll.
Leider scheint es, dass `laminas-api-tools` so gebaut ist. Üblicherweise wird der Service in den Controller injected. Der Controller kennt Entität-Typ und Collection-Typ (aus der Config). `laminas-api-tools` baut beides zusammen - weshalb wir nicht bemerken, dass das etwas seltsam gemacht wird.
Da wir hier den Service ohne den zugehörigen Controller verwenden, fehlt ein Teil vom Setup - weshalb ich es manuel nachhole..

Uncool - und ich muss mal schauen, ob es besser geht...